packages/text-field/index.tsx

```diff
@@ -321,32 +322,27 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
       textarea,
     } = this.props;
     const {foundation} = this.state;
-    const textField = (
-      <div
-        {...this.otherProps}
-        className={this.classes}
-        onClick={() => foundation && foundation.handleTextFieldInteraction()}
-        onKeyDown={() => foundation && foundation.handleTextFieldInteraction()}
-        key='text-field-container'
-      >
-        {leadingIcon ? this.renderIcon(leadingIcon, onLeadingIconSelect) : null}
-        {foundation ? this.renderInput() : null}
-        {label && !fullWidth ? this.renderLabel() : null}
-        {outlined ? this.renderNotchedOutline() : null}
-        {!fullWidth && !textarea && !outlined ? this.renderLineRipple() : null}
-        {trailingIcon ? this.renderIcon(trailingIcon, onTrailingIconSelect) : null}
-      </div>
-    );
 
-    if (helperText) {
-      return (
-        <React.Fragment>
-          {textField}
-          {this.renderHelperText()}
-        </React.Fragment>
-      );
-    }
-    return textField;
+    return (
+      <React.Fragment>
+        <div
+          {...this.otherProps}
+          className={this.classes}
+          onClick={() => foundation && foundation.handleTextFieldInteraction()}
+          onKeyDown={() => foundation && foundation.handleTextFieldInteraction()}
+          key='text-field-container'>
+          {leadingIcon ? this.renderIcon(leadingIcon, onLeadingIconSelect) : null}
+          {foundation ? this.renderInput() : null}
+          {label && !fullWidth ? this.renderLabel() : null}
+          {outlined ? this.renderNotchedOutline() : null}
+          {!fullWidth && !textarea && !outlined ? this.renderLineRipple() : null}
+          {trailingIcon ? this.renderIcon(trailingIcon, onTrailingIconSelect) : null}
+        </div>
+        {(helperText || false) && <div className={cssClasses.HELPER_LINE}>
```

Comment:
can you just add the `HELPER_LINE` element and the helperText null check into the `renderHelperText` method. This block of JSX can then just be:

```
{this.renderHelperText()}
```